mmstudio/src/main/java/org/micromanager/data/internal/multipagetiff/OMEMetadata.java

```diff
@@ -172,7 +172,7 @@ private void startSeriesMetadata(int seriesIndex, String baseFileName) {
       }
       metadata_.setStageLabelName(positionName, seriesIndex);
 
-      String instrumentID = MetadataTools.createLSID("Microscope");
+      String instrumentID = MetadataTools.createLSID("Microscope", 0);
```

Comment:
I think this specifically needs to be "Instrument" instead of "Microscope"

Reply:
Is there any documentation on keywords that are essential?  OME metadata have been opaque/baroque/verbose/incomprehensible to me for the last 20 years;)

Reply:
I usually just use the XSD: https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Instrument_ID

The "Pattern" is a regex match: `((urn:lsid:([\w\-\.]+\.[\w\-\.]+)+:\S+:\S+)|(\S+:\S+)) & ((urn:lsid:([\w\-\.]+\.[\w\-\.]+)+:Instrument:\S+)|(Instrument:\S+))`

Relevant bit here is "Instrument:{any-non-white-space-character}"